src/templates/landingv2.tsx

```
@@ -6,14 +6,20 @@
-export default function LandingPageV2({ data }: { data: TProject[] }) {
-  const [selectedCategory, setSelectedCategory] = useState('All');
+export default function LandingPageV2({
+  projects,
+  publishers,
+}: {
+  projects: TProject[];
+  publishers: TPublisher[];
+}) {
+  const [selectedCategory, setSelectedCategory] = useState("All");
   const [isStartProjectOpen, setIsStartProjectOpen] = useState(false);
+  const categories = projects.map((project) => (project.category));
```

ESLint

[error] Unexpected parentheses around single function argument having a body with no curly braces.

(style/arrow-parens)